src/events/messageCreate.ts

```diff
@@ -94,10 +94,17 @@ const convertResumePdfsIntoImages = async (
     return await message.channel.send('Resume must be 1 page.');
   }
 
+  const fileMatch = pdfLink.match('[^/]*$') || ['Resume'];
+  const fileName = fileMatch[0];
   // Convert the resume pdf into image
   const imgResponse = await convertPdfToPic('tmp/resume.pdf', 'resume', width * 2, height * 2);
-  // Send the image back to the channel
-  const preview_message = await message.channel.send({
+  // Send the image back to the channel as a thread
+  const botMessage = await message.channel.send('Converting Resume...');
```

Comment:
It would be good if the thread could be created from the message containing the PDF file, instead of from this newly sent message.

Reply:
I agree with this as well.